js/models/conversations.js

```diff
@@ -1409,13 +1409,14 @@
                   groupId: destination,
                 }
               );
-              await Promise.all(
-                members.map(async m => {
-                  const memberPubKey = new libsession.Types.PubKey(m);
-                  await libsession
-                    .getMessageQueue()
-                    .sendUsingMultiDevice(memberPubKey, mediumGroupChatMessage);
-                })
+
+              const rawMessage = libsession.Utils.MessageUtils.toRawMessage(
+                destinationPubkey,
+                mediumGroupChatMessage
+              );
+              await libsession.Sending.MessageSender.sendToMediumGroup(
```

Comment:
why are we using the message sender directly instead of making it go through the queue?

Reply:
The purpose of the queue was to wait for the session with the pubkey, which does not make sense if the pubkey is groupId.

Reply:
The queue was designed to have a dual purpose:
- Wait until we have a session (which isn't the case anymore, and we have checks in there to ignore session logic if we have a medium group message)
- Allow for messages to be sent at a later time in the case where user sends it and exits straight away. It also has the UI logic hooked onto it.

Reply:
Updated